types/nodemark/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "esModuleInterop": true,
+        "module": "commonjs",
+        "lib": [
+            "dom",
```

Comment:
why do you need this? if for `console.log`, just use TS lint assertions:
https://github.com/DefinitelyTyped/DefinitelyTyped#my-package-teststs

Reply:
removed 'dom' library from tsconfig

Reply:
I manually declare the console instead, so now it works in CI (static analysis without running). And it also work when run locally with ts-node calling the underneath js package.